src/client/clientenvironment.h

```diff
@@ -130,25 +131,26 @@ class ClientEnvironment : public Environment
 	// Get event from queue. If queue is empty, it triggers an assertion failure.
 	ClientEnvEvent getClientEnvEvent();
 
-	virtual void getSelectedActiveObjects(
-		const core::line3d<f32> &shootline_on_map,
-		std::vector<PointedThing> &objects
-	);
+	virtual void getSelectedActiveObjects(const core::line3d<f32> &shootline_on_map,
+			std::vector<PointedThing> &objects);
```

Comment:
Has the first param not been moved to a new line because both params together would cross the line length limit anyway?

Reply:
Yes